plexpy/webserve.py

```diff
@@ -4554,7 +4556,7 @@ def get_item_user_stats(self, rating_key=None, grouping=None, **kwargs):
 
         if rating_key:
             item_data = datafactory.DataFactory()
-            result = item_data.get_user_stats(rating_key=rating_key, grouping=grouping)
+            result = item_data.get_user_stats(rating_key=rating_key, media_type=media_type, grouping=grouping)
```

Comment:
Line break so it's cleaner.
```suggestion
            result = item_data.get_user_stats(rating_key=rating_key,
                                              media_type=media_type,
                                              grouping=grouping)
```